pkg-r/tests/testthat/test-shiny-app.R

```diff
@@ -9,69 +9,7 @@ test_that("app database example loads without errors", {
   test_app_file <- tempfile(fileext = ".R")
```

Comment:
Is the intention here that `pkg-r/tests/testthat/apps/basic/app.R` is the new `test_app_file`?